@vates/read-chunk/index.js

```diff
@@ -80,7 +81,15 @@ exports.readChunkStrict = async function readChunkStrict(stream, size) {
   }
 
   if (size !== undefined && chunk.length !== size) {
-    const error = new Error(`stream has ended with not enough data (actual: ${chunk.length}, expected: ${size})`)
+    // Buffer.isUtf8 is too recent for now
+    // @todo : replace external package by Buffer.isUtf8 when the supported version of node reach 18
+
+    // cut the message to a sane length and make it usable
+    const content = chunk.subarray(0, 128).toString(isUtf8(chunk) ? 'utf-8' : 'base64')
+
+    const error = new Error(
```

Comment:
If too complicated, another solution would be to add a enumerable property if the chunk is small and UTF-8.

I don't love this solution, but this would be acceptable to me if the first one is too hard.